### PR TITLE
Create response_headers.md

### PR DESCRIPTION
**Pull request by timpanogos**
To be linked from https://eveonline-third-party-documentation.readthedocs.io/en/latest/crest/index.html under "Calls" right above "Character"

### CODE REVIEW

File: docs/crest/response_headers.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Response Headers
+## Cache-Control Header
+
+In the XML API, the cached for time is reported in the XML body of the HTTP Response.  For Crest the cached for time (in seconds) is returned in a response header.
```
Comment (polarina) [inline docs/crest/response_headers.md]: The XML API should not be in the primary sentence of this paragraph. How about, at the end of the paragraph, you have a sentence along the lines of "This is in contrast to the XML API, where ..."?

**Comment by polarina** [comment]
Do you believe the Java code snipped is valuable information to potential readers?

**Comment by timpanogos** [comment]
@polarina agree, the snipped is likely noise, programmers will know how to get this.

**Comment by timpanogos** [comment]
leaves the page being nothing more than about a sentence

**Pull request closed without merging**